### PR TITLE
Tidy up email templates #5702

### LINKED ISSUE #5702: Tidy up email templates
Many rooms for tidying up such as:
- `<p/>` everywhere, but `<p>` is not a self-closing tag and even if it is, it is against our style guide to close it.
- HTML attributes not surrounded by quotes.
- No/extraneous EOL at the end of file.
- Missing punctuations/periods/space, weird line breaks, etc.

### PR DESCRIPTION
**Pull request by Neurrone**
Fixes #5702.

### CODE REVIEW

**Comment by Neurrone** [comment]
Ready to review.

**Comment by wkurniawan07** [comment]
Just a heads up, this might have a minor conflict with #6454.

File: src/test/resources/emails/sessionUnpublishedEmailForStudent.html [hunk 1]
```diff
@@ -1,16 +1,17 @@
-Hello student1 In Course1</td></div>'",
-<p/>
+<p>Hello student1 In Course1</td></div>'",</p>p>
```
Comment (ablyx) [inline src/test/resources/emails/sessionUnpublishedEmailForStudent.html]: typo

File: src/test/resources/emails/sessionUnpublishedEmailForStudent.html [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 
+<p></p>
```
Comment (ablyx) [inline src/test/resources/emails/sessionUnpublishedEmailForStudent.html]: is this needed

File: src/test/resources/emails/studentCourseJoinEmail.html [hunk 3]
```diff
@@ -1,22 +1,32 @@
-Hello Student Name,
+<p>Hello Student Name,</p>
 
-<p/>
-The course <i>Course Name</i> is using the TEAMMATES System to collect feedback.
+<p><p>
```
Comment (ablyx) [inline src/test/resources/emails/studentCourseJoinEmail.html]: additional `<p>`?

File: src/test/resources/emails/studentCourseRejoinAfterGoogleIdResetEmail.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
-Hello Student Name,
+<p>Hello Student Name,</p>
 
-<p/>
+<p><p/>
```
Comment (ablyx) [inline src/test/resources/emails/studentCourseRejoinAfterGoogleIdResetEmail.html]: Tag shouldnt be closed here

File: src/test/resources/emails/systemCrashReportEmail.html [hunk 5]
```diff
@@ -1,37 +1,35 @@
...
-{}
-<br>
+<p><b>Request Path</b>
+    <br>/page/studentHome
+</p>
 
-<b>Stack Trace</b>
-<pre><code>
+<p><b>Request Parameters</b>
+    <br>{}
```
Comment (ablyx) [inline src/test/resources/emails/systemCrashReportEmail.html]: should be `{}<br>`? I think the order matters

File: src/test/resources/emails/systemCrashReportEmailLessInfo.html [hunk 6]
```diff
@@ -1,37 +1,35 @@
...
-{}
-<br>
+<p><b>Request Path</b>
+    <br>/page/studentHome
+</p>
 
-<b>Stack Trace</b>
-<pre><code>
+<p><b>Request Parameters</b>
+    <br>{}
```
Comment (ablyx) [inline src/test/resources/emails/systemCrashReportEmailLessInfo.html]: same as above

File: src/test/resources/emails/systemCrashReportEmail.html [hunk 7]
```diff
@@ -47,5 +45,4 @@
 	at org.testng.TestRunner.privateRun(TestRunner.java:771)
 	at org.testng.TestRunner.run(TestRunner.java:621)
 	...
-
 </code></pre>
```
Comment (ablyx) [inline src/test/resources/emails/systemCrashReportEmail.html]: should have a closing tag `</p>`

File: src/test/resources/emails/systemCrashReportEmailLessInfo.html [hunk 8]
```diff
@@ -47,5 +45,4 @@
 	at org.testng.TestRunner.privateRun(TestRunner.java:771)
 	at org.testng.TestRunner.run(TestRunner.java:621)
 	...
-
 </code></pre>
```
Comment (ablyx) [inline src/test/resources/emails/systemCrashReportEmailLessInfo.html]: should have a closing tag `</p>`

File: src/test/resources/emails/sessionClosedEmailForStudent.html [hunk 9]
```diff
@@ -1,16 +1,14 @@
-Hello student1 In Course1</td></div>'",
+<p>Hello student1 In Course1</td></div>'",</p>
```
Comment (chowyb) [inline src/test/resources/emails/sessionClosedEmailForStudent.html]: Adding on to this review just to clarify a few things:

1. The `</td></div>'"` here is part of the course name used for testing, and is supposed to look like this. This should not be changed.
2. Usually there is no need to comment on every instance of a problem in the test HTML files, as these files are changed via [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md), and should not be changed directly. Therefore, repeated instances of the same issue in the test files are likely caused by the same piece of code that is being tested, and so adding a single comment to the original code is preferred (so there's no need to point out `</p>p>` in the test HTML files in this case, as they likely originate from the ones found in the main code)

**Comment by Neurrone** [comment]
Thanks @ablyx, will fix tomorrow.

**Comment by Neurrone** [comment]
@ablyx changes made.

**Comment by chowyb** [comment]
@Neurrone you will need to update this branch and fix the merge conflicts due to #6454. I'll review the code here once it achieves CI green.

**Comment by Neurrone** [comment]
@chowyb conflicts fixed.

File: src/main/resources/studentEmailFragment-courseJoin.html [hunk 10]
```diff
@@ -1,13 +1,22 @@
-<p/>
-The course <i>${courseName}</i> is using the TEAMMATES System to collect feedback.
+<p>
+    The course <em>${courseName}</em> is using the TEAMMATES System to
+    collect feedback.
+</p>
+<p>
+    <strong>To "join" the course, please go to this Web
+        address: </strong><a href="${joinUrl}">${joinUrl}</a> 
+        <br>*If prompted to log in, use your Google account to log in. If you do not
```
Comment (chowyb) [inline src/main/resources/studentEmailFragment-courseJoin.html]: Align this line to 4 spaces as well.

File: src/main/resources/studentEmailFragment-courseJoin.html [hunk 11]
```diff
@@ -1,13 +1,22 @@
...
+    have a Google account, please create one from the <a
+        href="https://accounts.google.com/NewAccount">Google
+        Accounts page</a>. 
+        <br>*The above link is unique to you. Please do not share it with your classmates.
```
Comment (chowyb) [inline src/main/resources/studentEmailFragment-courseJoin.html]: Align this line to 4 spaces as well.

File: src/main/resources/studentEmailFragment-googleIdReset.html [hunk 12]
```diff
@@ -1,18 +1,25 @@
...
+    for the course <i>${courseName}</i> to its previous Google account.
+</p>
 
-<p/>
-<strong>If you wish to continue to access TEAMMATES using your Google account for this course, please rejoin this course through this Web address: </strong><a href="${joinUrl}">${joinUrl}</a>
-<br>*If prompted to log in, use your Google account to log in. If you do not have a Google account, please create one from the <a href="https://accounts.google.com/NewAccount">Google Accounts page</a>. 
-<br>*The above link is unique to you. Please do not share it with your classmates.
+<p>
+    <strong>If you wish to continue to access TEAMMATES using your Google account for this course, 
+        please rejoin this course through this Web address: </strong>
```
Comment (chowyb) [inline src/main/resources/studentEmailFragment-googleIdReset.html]: There are currently two different ways of using the `<strong>` element in this file, one where `</strong>` is at the end of a line, and one where `</strong>` is on a line on its own. Standardising this across the files being modified such that `</strong>` is in a line on its own would be good.

**Comment by Neurrone** [comment]
@chowyb changes made.

* standardized <a> and <strong> usage so that they are on their own line
* fixed html in the ${instructor fragment} tag in EmailGenerator.java
* some other small fixes that i missed out earlier

**Comment by chowyb** [comment]
@Neurrone one instance of the `<strong>` element in `studentEmailFragment-googleIdReset.html` still needs fixing, but otherwise everything looks fine 👍

**Comment by Neurrone** [comment]
@chowyb fixed.

**Comment by wkurniawan07** [comment]
@Neurrone there is a minor merge conflict to resolve.

**Comment by Neurrone** [comment]
@wkurniawan07 conflict fixed.

**Pull request merged**